.github/workflows/build-test-release.yml

```diff
@@ -126,10 +126,6 @@ jobs:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
           args: --severity-threshold=high
-      - name: Upload result to GitHub Code Scanning
-        uses: github/codeql-action/upload-sarif@v1
-        with:
-          sarif_file: snyk.sarif
```

Comment:
Why are we deleting those lines?

Reply:
In every other snmp repository this step is being commented out, it's giving the error:

Reply:
PTAL https://github.com/splunk/addonfactory-ucc-generator/blob/5bfc6c389a49f8f74e1df30c97427794b79d6e9d/.github/workflows/build-test-release.yml#L158-L172

Reply:
```
        with:
          args: --sarif-file-output=snyk.sarif
```
lines are missing